src/mqt/ionshuttler/multi_shuttler/inside/graph.py

```
@@ -15,8 +16,10 @@ class Graph(nx.Graph):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)